packages/keyring-controller/src/KeyringController.ts

```diff
@@ -364,6 +364,12 @@ export type ExportableKeyEncryptor = GenericEncryptor & {
   importKey: (key: string) => Promise<unknown>;
 };
 
+type MutuallyExclusiveCallback<T> = ({
```

Comment:
Nit: A TSDoc description would be helpful here

Reply:
Added in [1295da2](https://github.com/MetaMask/core/pull/4192/commits/1295da23544db90c9bcba1c61c1971ac576c325a). I think we can keep it not exported though